packages/docs/src/components/layout/HeaderFullWidth.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import CMSLogoIcon from '../icons/CMSLogo';
+import SearchForm from '../content/SearchForm';
+
+const HeaderFullWidth = () => {
+  return (
+    <div className="ds-l-row ds-u-margin--0 ds-u-padding--2 ds-u-padding-left--0 ds-u-border-bottom--2">
+      <div className="ds-u-display--flex ds-l-col--6 ds-u-align-items--center ds-u-padding-left--0">
+        <CMSLogoIcon />
+        <h1 className="ds-u-padding-left--1 ds-text-heading--2xl ds-u-margin--0">Design System</h1>
```

Comment:
I'm not sure what modern best practice is here, but I'm used to making these site logos/headings links to the root of the site. Should we do that?